cfme/tests/containers/test_logging.py

```diff
@@ -55,7 +58,7 @@ def test_external_logging_activated(provider, appliance, test_item, get_ose_logg
     if test_item.obj is ContainersProvider:
         obj_inst = provider
     else:
-        obj_inst = test_item.collection_obj(appliance).get_random_instances().pop()
```

Comment:
This is the worst. 

I want to move away from using existing openshift objects to creating/deleting objects we need in the tests.